### ISSUE TITLE
module 'multiprocessing' has no attribute 'connection'

### ISSUE THREAD

**Issue by nandevers** [awslabs/aws-data-wrangler#577]
**Describe the bug**
Python 3.8.8
After the command line installation through: `conda install -c conda-forge awswrangler`. I got the following error.

```
>>> import awswrangler
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\83986\Anaconda3\envs\ds-recommendation\lib\site-packages\awswrangler\__init__.py", line 17, in <module>
    from awswrangler.pandas import Pandas  # noqa
  File "C:\Users\83986\Anaconda3\envs\ds-recommendation\lib\site-packages\awswrangler\pandas.py", line 45, in <module>
    class Pandas:
  File "C:\Users\83986\Anaconda3\envs\ds-recommendation\lib\site-packages\awswrangler\pandas.py", line 273, in Pandas
    def _read_csv_once_remote(send_pipe: mp.connection.Connection, session_primitives: "SessionPrimitives",
AttributeError: module 'multiprocessing' has no attribute 'connection'
``` 

**To Reproduce**
1. install via conda with a Python 3.8.8 environment
2. Import awswrangler into a session

After following the suggested workaround in this [question](https://stackoverflow.com/questions/60907299/error-import-awswrangler-attributeerror-module-multiprocessing-has-no-attrib), the import worked just fine.

**Comment by maxispeicher** [comment]
Are you trying to install `awswrangler` inside of the `base` environment? If that's the case you could try creating a new environment before running the `install` command like:
```bash
conda create --name awswrangler-env
conda activate awswrangler-env
conda install -c conda-forge awswrangler
```

The reasoning behind this is is that for me in the base environment `conda` installs a very old version of `awswrangler` in version `0.3.1`. However when running the install command in a separate environment it installs the latest version in `2.4.0`, which doesn't have the issue anymore.

**Comment by jaidisido** [comment]
I can confirm that the above works. Thanks @maxispeicher. Closing for now

**Comment by nandevers** [comment]
Hey guys. 

Actually I did't install it in the `base`, I had a dedicated environment for that. I am running `python 3.8.8`  which is def not the default python for the conda distribution. 

Anyway, the workaround seems to work for me. However, it is still a bug.

**Comment by maxispeicher** [comment]
My guess would be that there is some kind of dependency version clash in your environment which prevents installing the latest `awswrangler` version. Could you try explicitly installing version 2.4.0 and see if `conda` reports an error?

```
conda install -c conda-forge awswrangler
```

Also a list of the packages that you have installed in the environment would be helpful for tracking down the issue.

**Comment by nandevers** [comment]
Indeed, the following is the result, unless I made a mistake in the installation command:

```
conda install -c conda-forge awswrangler=2.4.0
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: |
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions
```

**Comment by maxispeicher** [comment]
The installation command looks fine 👍.
However `conda`s output is not really helpful. My guess would be some Version clash for `pandas` or `numpy`, just because they are the most probable for already existing in the environment. But that's impossible to know for sure, without knowing the installed packages in your environment. Maybe you could share the output of `conda list`.

**Comment by nandevers** [comment]
Guys, thanks a bunch for all the help. Turns out I am real dumb! The bug was in my set up and not in the `aws-data-wrangler` conda installation at all. So, PowerShell's execution policy was preventing the correct environment to be selected by VSCode. Then, everytime I started it out, it tried to run `aws-data-wrangler` on a different conda env, making both `numpy` and `pandas`, (and all others for that matter) diverge from the versions where `aws-data-wrangler` had been installed in the first place. 

In any case, I own up to causing a misreport of bug here. Sorry about that! Thank you all very much.
